contracts/KyberNetwork.sol

```diff
@@ -115,7 +116,8 @@ contract KyberNetwork is Withdrawable, Utils {
             for (uint i = 0; i < reserves.length; i++) {
                 if (reserves[i] == reserve) {
                     if (reserves.length == 0) return;
-                    reserves[i] = reserves[--reserves.length];
+                    reserves[i] = reserves[reserves.length - 1];
+                    --reserves.length;
```

Comment:
why break it into two lines? also, why x++ vs ++x. let's be consistent (I think x++ is nicer).
Only reason to use --x, is if we do it in one line.